CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -149,6 +164,8 @@ local --[[persistable:options_window_language_button]] function language_button(
 local --[[persistable:options_width_textbox_reset]] function width_textbox_reset() end
 local --[[persistable:options_height_textbox_reset]] function height_textbox_reset() end
 
+
+
```

Comment:
No, an oversight. I was going to write the callback function here, but I needed the closure.